frontend/src/services/objectService.ts

```diff
@@ -10,14 +11,14 @@ export default {
    * Post an object
    * @returns {Promise} An axios response
    */
-  createObject(object: any, bucketId?: string) {
+  createObject(object: any, bucketId?: string, axiosOptions?: AxiosRequestConfig) {
```

Comment:
Please remember to update the JSDocs for this function above - it has 3 `@params` that are missing